### PR TITLE
Update hcp kubevirt jobs to use odf 4.15

### PR DESCRIPTION
**Pull request by davidvossel**

### CODE REVIEW

**Comment by davidvossel** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-mce-disconnected periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal

**Comment by davidvossel** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-mce-disconnected periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal

**Comment by davidvossel** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-mce

**Comment by davidvossel** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-mce

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16__periodics.yaml [hunk 1]
```diff
@@ -148,7 +148,7 @@ tests:
       CCS_OPERATOR_PACKAGES: odf-operator,ocs-operator,mcg-operator,odf-csi-addons-operator
       HYPERSHIFT_NODE_COUNT: "2"
       LVM_OPERATOR_SUB_CHANNEL: stable-4.15
-      ODF_OPERATOR_SUB_CHANNEL: stable-4.14
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.15
```
Comment (tiraboschi) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16__periodics.yaml:151]: This value is used only to create a subscription pointing to that channel, in order to get it mirrored first please also update:
```
      CCS_OPERATOR_CHANNELS: stable-4.15,stable-4.15,stable-4.15,stable-4.15
      CCS_OPERATOR_PACKAGES: odf-operator,ocs-operator,mcg-operator,odf-csi-addons-operator
```

**Comment by davidvossel** [comment]
/close

inactive

**Pull request closed without merging**